collectors/CMPCollector.js

```diff
@@ -121,52 +123,67 @@ class CMPCollector extends BaseCollector {
         session.on('Runtime.executionContextCreated', async ({context}) => {
             // ignore context created by puppeteer / our crawler
             if (!context.origin || context.origin === '://' || context.auxData.type === 'isolated') {
+                if (context.auxData.type === 'isolated' && context.name.startsWith(ISOLATED_WORLD_PREFIX)) {
```

Comment:
Checking `context.auxData.type === ‘isolated’` twice here which is redundant. Do we need two `if`s? It’s difficult to understand the logic here.